litmus-portal/agent-plane-network-policies.yml

```diff
@@ -0,0 +1,90 @@
+## Policy for Self-agent Subscriber
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: subscriber-network-policy
+spec:
+  podSelector:
+    matchLabels:
+      app: subscriber
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
+    - from:
+        # gql-server --> self-agent, Accepting workflow requests from gql-server
+        - podSelector:
+            matchLabels:
+              component: litmusportal-server
+
```

Comment:
The ingress will be happening over the network? I believe we use the ingress path or the nodeport for communication even for self-agent

Reply:
For Ingress/NodePort, Users can use ipBlock. 
In namespace mode, we give service-name in SERVER_ADDR so this will help there. This is just for refrence default config, users will have to modify it a bit according to their config.